styleguide/assets/additional.scss

```diff
@@ -1,4 +1,5 @@
-@import 'server.css';
+@import url('https://nextcloud.github.io/server/apps/theming/css/default.css');
+@import url('https://nextcloud.github.io/server/core/css/server.css');
```

Comment:
This is auto deployed by github pages btw
https://github.com/nextcloud/server/settings/pages